libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -1548,7 +1552,12 @@ bool NavEKF3::using_extnav_for_yaw() const
 bool NavEKF3::configuredToUseGPSForPosXY(void) const
 {
     // 0 = use 3D velocity, 1 = use 2D velocity, 2 = use no velocity, 3 = do not use GPS
-    return  (sources.getPosXYSource() == AP_NavEKF_Source::SourceXY::GPS);
+    for (uint8_t i=0; i<num_cores; i++) {
+        if (sources.getPosXYSource(i) == AP_NavEKF_Source::SourceXY::GPS) {
```

Comment:
i think this should check only the active lane? otherwise AHRS will be very unhappy